profiles/common/modules/features/communities/communities.install

```diff
@@ -6,6 +6,23 @@
  */
 
 /**
+ * Implements hook_requirements().
+ */
+function communities_requirements() {
+  $requirements = [];
+  if (module_exists('nexteuropa_communities')) {
+    $t = get_t();
+    $args = array('%module' => 'Nexteuropa Communities');
+    $requirements['communities_module_conflict'] = [
```

Comment:
Don't use the short syntax for the array.
We try to follow the good practice of Drupal 7 on the platform.
see: https://www.drupal.org/docs/develop/standards/coding-standards#array for php 5.4.

Reply:
you right, nice spotted !

Fixed